src/config.rs

```diff
@@ -69,6 +69,9 @@ pub enum Config {
     #[strum(props(default = "0"))] // also change MediaQuality.default() on changes
     MediaQuality,
 
+    #[strum(props(default = "50"))]
```

Comment:
not sure about that default. googling a bit around with "max recipients smtp" or so, you easily find this limit, most times more, however. the largest group i am using has 45 ppl, afaik, this does not have problems (yet)